out/tables/reference_table_1/delta/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,6 @@
+{"protocol":{"minReaderVersion":1,"minWriterVersion":2}}
```

Comment:
Here the reader version is 1, but the in the Python file it was labelled as 2.

Reply:
Good point, should we even place the protocol version in the reference table model if it is anyways in the log? If it is, I should implement some checks that they match

Reply:
what do you think? Should the protocol version be part of the table model?

Reply:
Yes, it should be part of the table model. Two reasons:

 1. Test implemententors may wish to filter the tables based on the protocol version, only testing the versions they support. (Though preferable they should verify that they error on ones they don't support.)
 2. We should test that readers correctly detect when a table upgrades its protocol when enabling certain features.